backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/RunVmCommandBase.java

```diff
@@ -409,7 +409,8 @@ protected void addNumaPinningForDedicated(Guid vdsId) {
     }
 
     protected void setDedicatedCpus(VdsManager vdsManager) {
```

Comment:
I think we should start using "exclusively pinned" instead of "dedicated" otherwise the code will be quite confusing. E.g. this method could be named setExclusiveCpuPinning.